utilities/common.py

```diff
@@ -136,6 +136,18 @@ def common_parser_combine():
     parser.add_argument("--lumiScale", type=float, default=None, help="Rescale equivalent luminosity by this value (e.g. 10 means ten times more data and MC)")
     return parser
 
+def set_parser_default(parser, argument, newDefault):
+    # change the default argument of the parser, must be called before parse_arguments
+    f = next((x for x in parser._actions if x.dest ==argument), None)
+    if f:
+        logging.warning("")
+        logging.warning(f" >>> Modifying default of {f.dest} from {f.default} to {newDefault}")
```

Comment:
Noooooo no using >>> it interferes with finding git conflicts. Also not sure you really need a warning here if it works as intended, but if you want output, I'd use logger.info. For the case where it didn't work, logger.warning is appropriate, or even throwing an exception

Reply:
This comes from me, the idea of the warning was to make it always visible (with the colored logger it would be yellow instead of grey), because one might not be aware that the default is being changed inside the script if they only checked the original option (obviously this code is only used by us and by now we all know, but still ...)

Reply:
and the >>> was also a leftover from my old code ... my apologies

Reply:
I have removed the ">>>" now